policyengine_us/parameters/gov/states/al/tax/income/deductions/itemized/medical_expense.yaml

```diff
@@ -0,0 +1,12 @@
+description:  Medical expenses over this percentage of AGI are deductible from taxable income.
+  2021-01-01: 0.04
+
+metadata:
+  label:  Medical expenses over this percentage of AGI are deductible from taxable income.
+  period: year
+  unit: /1
+  reference:
+    - title: 2022 Schedule A (Form 1040)
+      href: https://www.revenue.alabama.gov/ultraviewer/viewer/basic_viewer/index.html?form=2023/01/22f40schabdc_blk.pdf
+    - title: 2022 Form 40 Alabama Income Tax Return
+      href: https://www.revenue.alabama.gov/ultraviewer/viewer/basic_viewer/index.html?form=2023/02/22f40.pdf
```

Comment:
please also use this source for all parameters (point to exact section of the law document where you can find the parameter) f.e. - here: (13)
https://law.justia.com/codes/alabama/2021/title-40/chapter-18/article-1/section-40-18-15/